odoo/addons/base/migrations/11.0.1.3/post-migration.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# © 2017 bloopark systems (<http://bloopark.de>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import json
+
+from odoo.tools import pickle
+
+from openupgradelib import openupgrade
+
+from psycopg2.extensions import AsIs
+
+
+def map_ir_actions_server_fields(cr):
```

Comment:
as written above, you need to do this for all ir.actions.* models. Don't rely on the link from the action to the ir.value record, do it the other way around via https://github.com/OCA/OCB/blob/10.0/odoo/addons/base/ir/ir_values.py#L95, possibly filtering for key and key2

Reply:
@hbrunn I addressed your comments in #1121 (except that ir.values' action_id is almost never filled and when it is, the key already provides the action reference as it does in all other instances)